app/helpers/works_helper.rb

```diff
@@ -0,0 +1,39 @@
+module WorksHelper
+  def list_works(category)
+    works_rows = ""
+    works_in_category_sorted = Work.where(category: category).sort_by { |work| work.votes.count }.reverse
```

Comment:
It's fine that you used a helper here, but this is large enough that it would be easier to read if it were a partial.